src/main/java/com/dev/movietheatre/annotation/PasswordValidator.java

```diff
@@ -0,0 +1,21 @@
+package com.dev.movietheatre.annotation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class PasswordValidator implements ConstraintValidator<PasswordConstraint, String> {
+    private String password;
+    private String repeatPassword;
+
+    public void initialize(PasswordConstraint constraint) {
+        this.password = constraint.password();
+        this.repeatPassword = constraint.repeatPassword();
+    }
+
+    public boolean isValid(String obj, ConstraintValidatorContext context) {
```

Comment:
I guess better to change the name of the variable

Reply:
done